### PR TITLE
#47 - Use discord webhooks and remove unnecessary notifications

### PR DESCRIPTION
**Pull request by TomBurch**

### CODE REVIEW

**Comment by TomBurch** [comment]
image

File: app/DiscordWebhook.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    }
+
+    public static function notifyStaff(string $content)
+    {
+        $client = new Client(['headers' => ['Content-Type' => 'application/json']]);
+        $response = $client->request('POST', config('services.discord.staff_webhook'), [
+            'json' => ['content' => $content],
+        ]);
+    }
+}
```
Comment (Freddo3000) [inline app/DiscordWebhook.php:24]: No newline ree

File: app/DiscordWebhook.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            'json' => ['content' => $content],
+        ]);
+    }
+
+    public static function notifyStaff(string $content)
+    {
+        $client = new Client(['headers' => ['Content-Type' => 'application/json']]);
+        $response = $client->request('POST', config('services.discord.staff_webhook'), [
+            'json' => ['content' => $content],
+        ]);
```
Comment (Freddo3000) [inline app/DiscordWebhook.php:22]: Could be nice to use embeds, though that might be better suited for future PRs.

File: resources/views/nav/notifications.blade.php [hunk 3]
```diff
@@ -1,18 +1,2 @@
...
-        @unless ($notifications->isEmpty())
-            <span class="notifications-has-unread pulse pulse-info"></span>
-        @endunless
-    </a>
-
-    <div class="dropdown-menu" style="width: 25rem;max-height: 40rem;overflow-y: auto;max-width: calc(100vw - 3rem);">
-        @include('nav.notifications-list', ['notifications' => $notifications])
-    </div>
-</li>
+</li>
```
Comment (Freddo3000) [inline resources/views/nav/notifications.blade.php:2]: No newline ree

**Review by Freddo3000 — APPROVED** [review]
Waiting on @BorderKeeper

File: app/DiscordWebhook.php (hunk 2 above)
Comment (TomBurch) [reply app/DiscordWebhook.php:22]: Once larvael can be updated there will be a better HTTP library so I'll look at doing more complicated requests then
alternativelyI might just route info to SvenBot and rework all the messages at that point

**Review by BorderKeeper — APPROVED** [review]
lgtm

**Comment by BorderKeeper** [comment]
Blocked on @WirelessPillow generating discord api keys.

**Pull request merged**